### PR TITLE
Fix: Provide missing hidden imports for astroid v. 1.4.1.

### PR DESCRIPTION
**Pull request by bjones1**
This fix makes pylint (and its tests) work.

### CODE REVIEW

**Comment by bjones1** [comment]
Oops. Looks like Linux does actually uses `pkg_resources`, while Windows doesn't. Plus, the `astroid` hook didn't exclude tests. Working on a fix...

**Comment by bjones1** [comment]
Fixed. This includes the commits in #1701, so I'm assuming that pull request should be closed? However, I'll ask @matysek to take care of that, since we've started a conversation there.

File: tests/functional/test_libraries.py [hunk 1]
```diff
@@ -102,7 +102,6 @@ def test_sphinx(tmpdir, pyi_builder, data_dir):
     # Note that including the data_dir fixture copies files needed by this test.
     pyi_builder.test_script('pyi_lib_sphinx.py')
 
-@pytest.mark.xfail(reason='pkg_resources is not supported yet.')
```
Comment (matysek) [inline tests/functional/test_libraries.py]: thanks for fixing pylint test.

**Comment by matysek** [comment]
Looks fine to me. Thanks.

File: tests/functional/test_libraries.py (hunk 1 above)
Comment (bjones1) [inline tests/functional/test_libraries.py]: My pleasure! Thanks for the hard work on PyInstaller. The fact that just dropping in a few extra datas defines and pkg_resources works shows how well-designed the program is.

**Pull request merged**